alerting/src/main/kotlin/com/amazon/opendistroforelasticsearch/alerting/resthandler/RestIndexMonitorAction.kt

```diff
@@ -174,7 +193,21 @@ class RestIndexMonitorAction(
             }
         }
 
+        private fun onUpdateMappingsResponse(response: AcknowledgedResponse) {
+            if (response.isAcknowledged) {
+                log.info("Updated ${ScheduledJob.SCHEDULED_JOBS_INDEX} with mappings.")
+                scheduledJobIndexUpdated = true
+                prepareMonitorIndexing()
+            } else {
+                log.error("Updated ${ScheduledJob.SCHEDULED_JOBS_INDEX} mappings call not acknowledged.")
+                channel.sendResponse(BytesRestResponse(RestStatus.INTERNAL_SERVER_ERROR,
+                        response.toXContent(channel.newErrorBuilder(), EMPTY_PARAMS)))
```

Comment:
Same thing here. Lets provide an error message.

Reply:
Will change it